src/code/title_setup.c

```diff
@@ -1,7 +1,7 @@
+#include "console_logo_state.h"
 #include "setup_state.h"
 
 #include "global.h"
-#include "console_logo_state.h"
 
```

Comment:
doesn't have to change now but for the sake of preventing edit wars, I think the order of `#include`s should be (matching typical C codebases style)
```c
#include "header corresponding to the .c"

#include "libc" // sorted alphabetically
#include "libc64, libu64, ultralib" // sorted alphabetically
#include "other headers" // sorted alphabetically

#include "global.h" // garbage headers / temporary includes
```
For example:
```c
#include "setup_state.h"

#include "stdbool.h"
#include "console_logo_state.h"
#include "z64game.h"

#include "global.h"
```

Reply:
idk I've been on the fence about it but I think I'd prefer not hoisting the "primary" header to the top and just have it all sorted alphabetically.

Other than that, I think I agree (assuming no .bss issues crop up). I've mostly just been sorting by folder name alphabetically, then header name alphabetically, so that's why libc has been mixed in with the other header. maybe `-Iinclude/libc` should just be dropped so that it's easier to sort. 

And just to make sure you're aware, I have been moving `assets/` headers after "global.h" because object headers don't currently include the proper types.

Reply:
I would like primary header(s) on top. Plural for, for example, file_select_state (gamestate) and file_select (internal header for the 3 C files)

Reply:
idk I just think it adds an extra unnecessary level of micromanagement, and I just find it looks cleaner to have everything alphabetically sorted. With VSCode you can already find what you need fairly fast, and except for the headers that have already been renamed from expected og, you can just search the source file name to find the proper header.